src/pkgs/files/login.nix

```diff
@@ -7,14 +7,20 @@ writeScript "login" ''
   #!/system/bin/sh
   set -e
 
-  export USER=$(/system/bin/whoami)
+  export USER=nix-on-droid
```

Comment:
I'm afraid of a scenario where the android user*name* somehow leaks into the nix-on-droid environment and doesn't get mapped to anything meaningful. There was a reason why I did bother with all that stuff, but I didn't track history in git back then, so I cannot exactly recall my reasoning behind that.

Reply:
If you want to get rid of the Android-given username that much, you should probably also remove passing it to the .login-inner below.

Reply:
Why do you think it would be needed to remove passing the username to login-inner? That way I would need to hardcode the name twice and login-inner gets called only from login script.

Reply:
If `/etc/passwd`/`/etc/group` generation has to stay here, than yes, you'll need to hardcode it twice.
If it could be moved into the initial setup, then there would be no need to have it exported in the outer login at all.

Now the question is what made me do that in the outer login back then.